docs/objects.md

```diff
@@ -328,6 +328,195 @@ The formula to derive the pool's PKIprofile is as follows:
 ```
 pkiprofilenam = poolName + "-pkiprofile"
 ```
+### Markers for Avi Objects
+
+With Avi controller version 20.1.5, Avi objects can be labelled with markers. Each marker is pair of `key` : `<value1>`, `<value2>`. Each object can have multiple markers. Markers can be used to group objects at Avi controller side and provide permission to users to access these objects. Avi controller support markers to all license types from 20.1.6.
+
+With AKO version 1.5.1, AKO add markers to Avi objects. Each objects will be marked with fields of ingress/routes such as namespace, fqdn of ingress/route, service name of ingress/route etc. This will help in identifying object associate with specific ingress/route at controller side. Markers will be added to objects in SNI and EVH deployment modes.
```

Comment:
From AKO 1.5.1, markers would be added to all the AKO created Avi Objects.

Each object will be marked with fields such as OpenShift/Kubernetes namespaces to which the object belongs, fqdn present in the ingress/route objects, service names present in the ingress/route objects.